apps/full-site-editing/package.json

```diff
@@ -81,6 +85,7 @@
 		"classnames": "2.2.6",
 		"lodash": "*",
 		"moment": "*",
-		"newspack-blocks": "github:Automattic/newspack-blocks#v1.2.1"
+		"newspack-blocks": "github:Automattic/newspack-blocks#v1.3.1",
+		"swiper": "4.5.1"
```

Comment:
I think this is extraneous. Since we have `newspack-blocks` as a dependency, all its dependencies will also be installed automatically and swiper is there.

Reply:
#41669